README.md

```diff
@@ -252,3 +252,5 @@ file](./LICENSE).
 > http://www.apache.org/licenses/LICENSE-2.0
 >
 > Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
```

Comment:
Will this render correctly?

Reply:
See: https://github.com/pellared/splunk-otel-ruby/tree/patch-1

Reply:
Great, thanks!